File: Logs/logs.py
#!/usr/bin/env python
import pika
import time

def db_log(message):
	#Write log to local file
	stamp = time.asctime( time.localtime(time.time()))
	message = stamp + " -> " + message + "\n"
	
	with open('./server_log', 'a') as log_file:
		log_file.write(message)
	
	#Send message to DB for logging
	cred = pika.PlainCredentials('RMQ','RMQ_1234')

	connection = pika.BlockingConnection(pika.ConnectionParameters('192.168.1.5', 5672, '/', cred))
	channel = connection.channel()
	
	channel.exchange_declare(exchange='DB_logs', exchange_type='direct', durable=True)
	channel.basic_publish(exchange='DB_logs', routing_key='', body=message)

	connection.close()
	

File: Logs/logs.php
<?php

require_once __DIR__ . '/vendor/autoload.php';
use PhpAmqpLib\Connection\AMQPStreamConnection;
use PhpAmqpLib\Message\AMQPMessage;

function log_db($message) {
	//Create timestamp for log
	$message = date('D M d H:i:s Y', strtotime("now")) . " -> " . $message; 

	//Write log to local file
	$file = './server_log';
	$current = file_get_contents($file);
	$current .= "$message\n";
	file_put_contents($file, $current);
	
	//Send message to DB for logging
	$connection = new AMQPStreamConnection('192.168.1.5', 5672, 'RMQ', 'RMQ_1234');
	$channel = $connection->channel();

	$channel->exchange_declare('DB_logs', 'direct', false, true, false);

	$msg = new AMQPMessage($message);

	$channel->basic_publish($msg, 'DB_logs');

	$channel->close();
	$connection->close();

}

?>
